app/components/super_selector_component.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class SuperSelectorComponent < SelectorComponent
```

Comment:
What is a Super Selector?

Reply:
It's a selector that extend the `SelectorComponent` but adds a search input box, and display differently the chosen items:
<img width="285" alt="Capture d’écran 2022-03-29 à 13 45 33" src="https://user-images.githubusercontent.com/296452/160604292-11365e97-5fbb-4d6b-ab32-a7c3ee3850df.png">
<img width="269" alt="Capture d’écran 2022-03-29 à 13 45 40" src="https://user-images.githubusercontent.com/296452/160604301-3894575a-a990-42c9-a493-55e53f0cb0dd.png">
<img width="270" alt="Capture d’écran 2022-03-29 à 13 45 52" src="https://user-images.githubusercontent.com/296452/160604317-4eebc7ea-92aa-40fd-9bbd-84e0f1b6cb53.png">

Reply:
Done, and renamed to `SelectorWithFilterComponent` as it's more a filtering rather than a search.